docs/docs/external/getting-started/interop/asset-transfer.md

```diff
@@ -126,3 +128,182 @@ Assuming that the `simpleassettransfer` chaincode has been deployed in both netw
    ```bash
    ./scripts/getTokenBalance.sh network2 bob
    ```
+
+## 2. Corda with Corda
+One Corda network transfers either a bond or some tokens owned by `PartyA` (`CORDA_PORT=10006`) to `PartyB` (`CORDA_PORT=30009`) in the other network.
+
+### **Transfer or recover token (fungible) assets**
+Assuming that the corDapp `cordaSimpleApplication` has been deployed in both networks.
+- Navigate to `samples/corda/corda-simple-application` folder.
+- Create a `network-id` for each Corda network. This is a network state, and will be available in the vault of all the parties which are members of the network (if required, run the command `./clients/build/install/clients/bin/clients transfer get-party-name` with `CORDA_PORT=10006` or `CORDA_PORT=30006` or `CORDA_PORT=30009` to fetch the name of the parties `PartyA` in `Corda_Network` or `PartyA` in `Corda_Network2` or `PartyB` in `Corda_Network2` respectively).
+  ```bash
+  CORDA_PORT=10006 ./clients/build/install/clients/bin/clients network-id create-state "Corda_Network" -m "O=PartyA,L=London,C=GB"
+  CORDA_PORT=30009 ./clients/build/install/clients/bin/clients network-id create-state "Corda_Network2" -m "O=PartyA,L=London,C=GB;O=PartyB,L=London,C=GB"
+  ```
+  The above assumes that `Corda_Network` was started earlier with `1-node`, and `Corda_Network2` was started earlier with `2-nodes` (network state created in `Corda_Network` and `Corda_Network2` can be cross checked by running the commands `CORDA_PORT=10006 ./clients/build/install/clients/bin/clients network-id retrieve-state-and-ref` and `CORDA_PORT=30009 ./clients/build/install/clients/bin/clients network-id retrieve-state-and-ref` respectively in these networks).
```

Comment:
I think we can safely say corda networks were started with 2-nodes, and all 4-nodes are different parties (Since our script starts both networks with same profile, and its easier to do so also). So that by story, no common party is present in both networks.

Reply:
Done.